app/src/services/sync.js

```diff
@@ -159,22 +159,18 @@ const service = {
 
         // Case: already synced - record & update public status as needed
         if (comsObject) {
-          if (s3Public === undefined || s3Public === comsObject.public) {
-            response = await objectService.update({
-              id: comsObject.id,
-              userId: userId,
-              lastSyncedDate: new Date().toISOString()
-            }, trx);
-          } else {
-            response = await objectService.update({
-              id: comsObject.id,
-              userId: userId,
-              path: comsObject.path,
-              public: s3Public,
-              lastSyncedDate: new Date().toISOString()
-            }, trx);
-            modified = true;
-          }
+
+          response = await objectService.update({
+            id: comsObject.id,
+            userId: userId,
+            path: comsObject.path,
+            // update if public in s3
+            public: s3Public ? s3Public : comsObject.public,
+            lastSyncedDate: new Date().toISOString()
+          }, trx);
+
+          // if public flag changed mark as modified
```

Comment:
this line os not important.
marking the object as `modified` does nothing. 
we had intended to use the modified flag to determine if versions needed syncing as well.. but we currently always syn the versions (so long as an object exists)